### DISCUSSION TITLE
May we please have ability to adjust length of panel to be shorter than stretching from edge to edge of screen like Xfce panel option?

### DISCUSSION THREAD

**Discussion by Logansfury** [linuxmint/discussions#394] [Ideas]
Hello,

I primarily use Cinnamon, but I have seen that users of Xfce can right click their default panels, select preferences, and move a slider that reduces the length of the panel from screen edge to screen edge to a short horizontal bar in the middle of the screen just long enough for the number of icons in it.

I would like to request this be made available for Cinnamon as well please.

**Comment by JosephMcc** [comment]
https://github.com/orgs/linuxmint/discussions/216
